migrations/versions/4da21f1d702a_.py

```diff
@@ -0,0 +1,40 @@
+"""empty message
+
+Revision ID: 4da21f1d702a
+Revises: 1a45abbc3682
+Create Date: 2015-09-21 23:19:45.737008
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '4da21f1d702a'
+down_revision = '1a45abbc3682'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.create_unique_constraint(None, 'role_users', ['user_id', 'role_id'])
+    op.create_unique_constraint(None, 'user_expertise_domains', ['user_id', 'name'])
+    op.create_unique_constraint(None, 'user_languages', ['user_id', 'locale'])
+    op.create_unique_constraint(None, 'user_skills', ['user_id', 'name'])
+    op.alter_column('users', 'deployment',
+               existing_type=sa.VARCHAR(),
+               nullable=False)
```

Comment:
This appears to be the result of an auto-detected change by alembic:

```
INFO  [alembic.autogenerate.compare] Detected NOT NULL on column 'users.deployment'
```

I find it confusing, though, since the `deployment` field has always been not-nullable since its initial introduction in 79a51df098667fcac5e3aa7d295ea739368fc519. I can remove it from the migration if you want, though if that's the case, I'd like to figure out why alembic is detecting it so I won't have to constantly remove it from the auto-generated migrations going forward.